examples/3d/3d_scene.rs

```diff
@@ -1,19 +1,33 @@
 use bevy::prelude::*;
+use bevy_internal::{
+    render::wireframe::{Wireframe, WireframeConfig, WireframePlugin},
+    wgpu::{WgpuFeature, WgpuFeatures, WgpuOptions},
+};
 
 fn main() {
     App::build()
         .insert_resource(Msaa { samples: 4 })
+        .insert_resource(WgpuOptions {
```

Comment:
Can we break this out into its own `wireframe.rs` example? I want to keep `3d_scene.rs` simple

Reply:
Done.